corehq/apps/custom_data_fields/migrations/0007_regex_field_validation_privilege.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management import call_command
+from django.db import migrations
+
+from corehq.apps.accounting.models import SoftwarePlanEdition
+from corehq.privileges import REGEX_FIELD_VALIDATION
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def _grandfather_regex_field_validation_priv(apps, schema_editor):
+    call_command('cchq_prbac_bootstrap')
+
+    # Regex Field Validation is Pro Plan and higher
+    skip_editions = ','.join((
+        SoftwarePlanEdition.PAUSED,
+        SoftwarePlanEdition.COMMUNITY,
+        SoftwarePlanEdition.STANDARD,
+    ))
+    call_command(
+        'cchq_prbac_grandfather_privs',
+        REGEX_FIELD_VALIDATION,
+        skip_edition=skip_editions,
+        noinput=True,
+    )
+
+
+class Migration(migrations.Migration):
```

Comment:
This should have gotten created in `accounting` app not in `custom_data_fields`. https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/accounting/README.md#adding-a-new-privilege

Reply:
Thank you for pointing that out. I was unaware that all privilege toggles went to accounting and opted to put it in the app that was using the toggle. I have corrected this and moved it into the correct directory.